app/src/main/java/com/sun/tino/pomobox/MainActivity.java

```diff
@@ -1,13 +1,94 @@
 package com.sun.tino.pomobox;
 
+import android.support.annotation.NonNull;
+import android.support.design.widget.NavigationView;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.WindowManager;
 
-public class MainActivity extends AppCompatActivity {
```

Comment:
Nếu ứng dụng dùng nhiều Activity or Fragment nên tạo lớp Base cho activity or fragment cho toàn bộ ứng dụng.

Reply:
nhiều file change quá nên định để pull sau ms viết base